ci/main.sh

```diff
@@ -32,11 +32,22 @@ prepare_build_prerequisites() {
 prepare_test_env() {
   prepare_build_tool_env
 
-  export LD_LIBRARY_PATH="${GPG_INSTALL}/lib:${BOTAN_INSTALL}/lib:${JSONC_INSTALL}/lib:${RNP_INSTALL}/lib:$LD_LIBRARY_PATH"
+  export LD_LIBRARY_PATH="${GPG_INSTALL}/lib:${BOTAN_INSTALL}/lib:${JSONC_INSTALL}/lib:${RNP_INSTALL}/lib:${LD_LIBRARY_PATH-}"
```

Comment:
Would an improvement be:

```bash
export ...="...:${RNP_INSTALL}/lib${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}"
```

so that if `$LD_LIBRARY_PATH` is an empty string, there would not be a trailing colon?

Reply:
If $LD_LIBRARY_PATH is unset the script won't fail

Reply:
> If $LD_LIBRARY_PATH is unset the script won't fail

In which case `$LD_LIBRARY_PATH` would end up having a trailing `:` at the end.  I'm not familiar with the behaviour of `$LD_LIBRARY_PATH` with empty components, but if it's what's commonly accepted then I'm fine with it :)